satsense/features/rid.py

```diff
@@ -7,15 +7,14 @@
 import cv2
 import numpy as np
 from matplotlib import pyplot as plt
+from pysal.esda.getisord import G_Local
+from pysal.weights.Distance import DistanceBand
+from satsense.image import SatelliteImage
```

Comment:
Please use `isort` to sort imports

Reply:
reran isort on the file, shuffled it, reran it again, always gives the same output as the one that is in the repo...